client/buildpacks.go

```diff
@@ -125,16 +127,8 @@ func (b *buildpacksFunctionServer) run() (func(), error) {
 	if err != nil {
 		return nil, err
 	}
-
-	args := []string{"docker", "run",
-		"--network=host",
-		// TODO: figure out why these aren't getting set in the buildpack.
-		"--env=FUNCTION_TARGET=" + b.target,
-		"--env=FUNCTION_SIGNATURE_TYPE=" + b.funcType,
-		image,
-	}
-	cmd := exec.Command(args[0], args[1:]...)
-
+	var envVars = b.getRuntimeVars()
```

Comment:
based off usage, it seems like this should be `getRuntimeEnvVars`? It's unclear why "run" is included in the function, but "docker" is not

Reply:
I will include "docker" as well.